src/v2/view-builder/internals/BaseFooter.js

```diff
@@ -40,6 +40,10 @@ export default View.extend({
    */
   hasBackToSignInLink: true,
 
+  showFactorPageCustomLink: function() {
```

Comment:
I may think to utilize `links` instead of creating another flag in BaseFooter.
- Creating link is usually separated from Footer. see `utils/LinksUtils.js`. The logic to determine creating link seems to be like this ?
```
!this.options.appState.get('isPasswordRecovery')
&& ( currentForm === FORMS.CHALLENGE_AUTHENTICATOR || SelectAuthenticateVerify ) 
```
- Then override `links` from BaseFooter to determine whether to add that link or not.

Reply:
Agree with @haishengwu-okta ,